### PR TITLE
Feature/fraud indicators

### PR DESCRIPTION
**Pull request by colin99d**
Added the Zscore bankruptcy measure to the list of fraud measures.

### CODE REVIEW

File: gamestonk_terminal/fundamental_analysis/av_view.py [hunk 1]
```diff
@@ -626,21 +635,37 @@ def mscore(other_args: List[str], ticker: str):
...
+        print("Mscore Sub Stats:")
         for rkey, value in ratios.items():
             if rkey != "MSCORE":
                 print("  ", f"{rkey} : {value:.2f}")
 
         print(
-            "\n", "MSCORE: ", f"{ratios['MSCORE']:.2f} ({chance} chance of fraud)", "\n"
+            "\n" + "MSCORE: ",
+            f"{ratios['MSCORE']:.2f} ({chanceM} chance of fraud)",
+            "\n",
```
Comment (jmaslek) [inline gamestonk_terminal/fundamental_analysis/av_view.py:665]: Lets ditch this newline between the 2 ratios

**Comment by jmaslek** [comment]
Nice work again!  I find this funny for AAPL:

`ZSCORE:  -0.60 (high chance of bankruptcy) `

**Comment by colin99d** [comment]
Got those updated in! And from the research studies I read its 80-90% effective. So I guess its time to short $AAPL.

**Review by jmaslek — APPROVED** [review]

**Review by DidierRLopes — APPROVED** [review]
Nice work!

**Pull request merged**